test/check-machines-settings

```diff
@@ -599,8 +599,31 @@ class TestMachinesSettings(VirtualMachinesCase):
 
         b.wait_in_text("#vm-subVmTest1-watchdog-state", "none")
 
+        # General checks for watchdog
+        def openWatchDogDialog():
+            b.click("#vm-subVmTest1-watchdog-button")
+            b.wait_visible("#vm-subVmTest1-watchdog-modal")
+
+        def closeWatchDogDialog(selector):
+            b.click(selector)
+            b.wait_not_present("#vm-subVmTest1-watchdog-modal")
+
+        # Cancel
+        openWatchDogDialog()
+        closeWatchDogDialog("#watchdog-dialog-apply + button")
+
+        # "X" of the dialog
+        openWatchDogDialog()
+        closeWatchDogDialog("#vm-subVmTest1-watchdog-modal button[aria-label=Close]")
+
+        # Click "Add" without selecting action
+        openWatchDogDialog()
+        b.click("#watchdog-dialog-apply")
+        b.wait_in_text("#vm-subVmTest1-watchdog-modal .pf-m-danger", "unsupported configuration")
+        closeWatchDogDialog("#vm-subVmTest1-watchdog-modal button[aria-label=Close]")
+
```

Comment:
> But, by default, no selection in the watch dog dialog if a VM without watchdog device, do you think it is necessary for our test?

You mean that when no watchdog is specified, the dialog has no radio button selected?
![Screenshot from 2023-04-12 20-02-59](https://user-images.githubusercontent.com/42733240/231545678-e0fbc844-bf5c-4580-909a-126ff507f327.png)

Yeah, we can test that. But I think we should not test that using pixel tests. Pixel tests are usually meant to check that layout, spacing, colours... do not unexpectedly change. But that something is selected/not selected by default should be check in code.

You can test that no single radio button is selected with something like this. This gets a boolean of whether radio button for "Reset" is unchecked:
` b.assertFalse(b.get_checked("#vm-subVmTest1-watchdog-modal #reset"))`


```suggestion
        openWatchDogDialog()
        
        b.assertFalse(b.get_checked("#vm-subVmTest1-watchdog-modal #reset"))
        b.assertFalse(b.get_checked("#vm-subVmTest1-watchdog-modal #poweroff"))
        b.assertFalse(b.get_checked("#vm-subVmTest1-watchdog-modal #inject-nmi"))
        b.assertFalse(b.get_checked("#vm-subVmTest1-watchdog-modal #pause"))
        
        b.click("#watchdog-dialog-apply")
        b.wait_in_text("#vm-subVmTest1-watchdog-modal .pf-m-danger", "unsupported configuration")
        closeWatchDogDialog("#vm-subVmTest1-watchdog-modal button[aria-label=Close]")
        
```

And then we no longer need pixel_test_tag="without-watchdog" pixel test

Reply:
OK, will modify it, thanks